src/definitions/modules/calendar.js

```diff
@@ -19,6 +19,20 @@ window = (typeof window != 'undefined' && window.Math == Math)
     : Function('return this')()
 ;
 
+//support existing momentjs or date-fns library
+Date.prototype.getWeekOfYear = Date.prototype.week || Date.prototype.getWeek || function() {
```

Comment:
Let's move this function into the get methods on line [537](https://github.com/fomantic/Fomantic-UI/blob/3f9852142e61ef1f051ce1411c12d74803dda4e8/src/definitions/modules/calendar.js#L537)

Reply:
Done in latest commit